nixos/modules/misc/ids.nix

```diff
@@ -339,6 +339,7 @@
       rss2email = 312;
       cockroachdb = 313;
       zoneminder = 314;
+      colord = 315;
```

Comment:
I don't think this needs a static uid

Reply:
So you're looking for me to use `DynamicUser`?

I thought to do that but upstream provides a `tmpfiles.d` configuration and using that makes it unnecessary.

Reply:
Nah, just a standard user/group without any `uid`/`gid` setting